src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -141,23 +141,11 @@ This package is an internal implementation of the .NET Core SDK and is not meant
 
       <AspNetCoreReferenceDocXml Include="@(_ResolvedProjectReferencePaths->WithMetadataValue('IsReferenceAssembly', 'false')->'%(RootDir)%(Directory)%(FileName).xml')" />
       <AspNetCoreReferenceDocXml Include="@(_SelectedExtensionsRefAssemblies->'$(MicrosoftInternalExtensionsRefsPath)%(FileName).xml')" />
-    </ItemGroup>
 
-    <RepoTasks.GenerateSharedFrameworkDepsFile
-      DepsFilePath="$(ProjectDepsFilePath)"
-      TargetFramework="$(TargetFramework)"
-      FrameworkName="$(TargetingPackName)"
-      FrameworkVersion="$(ReferencePackSharedFxVersion)"
-      References="@(AspNetCoreReferenceAssemblyPath)"
-      RuntimeIdentifier="$(TargetRuntimeIdentifier)"
-      RuntimePackageName="$(PackageId)"
-      PlatformManifestOutputPath="$(ReferencePlatformManifestOutputPath)" />
```

Comment:
Why does this get removed?

Reply:
It looks to be called from Microsoft.AspNetCore.App.Runtime.csproj: https://github.com/aspnet/AspNetCore/blob/20fc1adf2ada01a4a1fc7934706d2b641acbb9f4/src/Framework/src/Microsoft.AspNetCore.App.Runtime.csproj#L230. As I don't think the result of GenerateSharedFrameworkDepsFile is being used for the Runtime project, should we move it all to the Ref project?

Reply:
> It looks to be called from Microsoft.AspNetCore.App.Runtime.csproj

That's correct, we use the platform manifest generated by the Runtime pack since that's what the file should refer to. The runtime pack uses `ReferenceCopyLocalPaths` to calculate what should be included, so I guess it should stay in that file. I'll add a comment explaining why it's there and what it's for.